### DISCUSSION TITLE
Tailwind CSS support

### DISCUSSION THREAD

**Discussion by rangelkoli** [remarkablemark/html-react-parser#1460] [Q&A]
Some classes of tailwind css are not being rendered properly, like hover classes and bg-black classes like that, is there any way to provide it with the tailwind css library?

**Comment by remarkablemark** [comment]
Can you provide a reproducible example with [StackBlitz](https://stackblitz.com/edit/html-react-parser)?

**Reply by rangelkoli** [reply]
Apologies, I was able to fix it, just had not setup Tailwind CSS properly

**Reply by remarkablemark** [reply]
Nice! Closing discussion as resolved.
